packages/agoric-cli/src/commands/inter.js

```diff
@@ -162,48 +308,101 @@ For example:
         readLatestHead(`published.vaultFactory.manager${opts.manager}.quotes`),
       ]);
       const info = fmtMetrics(metrics, quote, values(agoricNames.vbankAsset));
-      stdout.write(JSON.stringify(info, bigintReplacer, 2));
-      stdout.write('\n');
+      show(info, true);
     });
 
   const bidCmd = interCmd
     .command('bid')
     .description('auction bidding commands');
 
-  bidCmd
-    .command('by-price')
-    .description('Print an offer to bid collateral by price.')
-    .requiredOption('--price [number]', 'bid price', Number)
-    .requiredOption('--giveCurrency [number]', 'Currency to give', Number)
-    .requiredOption(
-      '--wantCollateral [number]',
-      'Collateral expected for the currency',
-      Number,
-    )
-    .option('--collateralBrand [string]', 'Collateral brand key', 'IbcATOM')
-    .option('--offerId [number]', 'Offer id', String, `bid-${now()}`)
+  /**
+   * @param {string} from
+   * @param {import('@agoric/smart-wallet/src/offers.js').OfferSpec} offer
+   * @param {Awaited<ReturnType<rpcTools>>} tools
+   */
+  const placeBid = async (from, offer, tools) => {
+    const { networkConfig, agoricNames, pollOffer } = tools;
+    const io = { ...networkConfig, execFileSync, delay, stdout };
+
+    const { home, keyringBackend: backend } = interCmd.opts();
+    const result = await sendAction(
```

Comment:
`sendAction` should come in from "tools" with configuration abstracted. non-blocking